movie-mlops-project/serving/streamlit_app.py

```
@@ -0,0 +1,515 @@
+        font-weight: bold;
+        font-size: 1.1rem;
+    }
+    .poster-img {
+        border-radius: 8px;
+        box-shadow: 0 2px 4px rgba(0,0,0,0.2);
+    }
+</style>
+""", unsafe_allow_html=True)
+
+# 환경변수 로드
+import os
```

Ruff (0.12.2)

Redefinition of unused `os` from line 10

Remove definition: `os`

(F811)